### DISCUSSION TITLE
Partial method doesn't work on task function

### DISCUSSION THREAD

**Discussion by lrahal** [apache/airflow#24073] [Q&A]

**Comment by josh-fell** [comment]
Can you provide more information on what "doesn't work"? Maybe even a code snippet of what you are trying to execute? Any additional context at all would be substantially useful.

Presuming you are using Airflow 2.3, [`partial()` is allowed on TaskFlow functions](https://airflow.apache.org/docs/apache-airflow/stable/concepts/dynamic-task-mapping.html#constant-parameters).
